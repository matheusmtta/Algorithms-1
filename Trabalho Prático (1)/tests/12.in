12 i
30 i
5 i
27 i
16 i
9 i
5 r
9 r
24 i
40 i
48 p
15 i
40 r
24 r
90 p
16 r
27 r
15 r
246 p
78 p
31 i
62 p
31 r
6 p
300 p
29 i
35 i
61 p
30 r
14 i
18 p
35 r
29 r
1 i
60 p
22 i
7 i
22 r
1 r
36 i
24 p
7 r
48 p
14 r
41 i
46 p
8 i
36 r
69 p
41 r
25 i
48 p
28 p
26 i
37 p
18 i
13 i
39 i
32 i
18 r
12 r
8 r
32 r
39 r
25 r
