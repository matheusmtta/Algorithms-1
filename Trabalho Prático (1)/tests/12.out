2
3
10
4
2
3
10
4
3
5
2
2
3
4
4
3
2
